torquedata/config.py.tmpl

```diff
@@ -7,9 +7,12 @@ POSTGRES_DB_PASSWORD="__DATABASE_PASSWORD__"
 # https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-SECRET_KEY
 SECRET_KEY="__RANDOM_SECRET_KEY__"
 
+# A list of jinja2 extensions to enable when rendering templates
```

Comment:
I think the only thing I'd need here is an example, or a link to somewhere with what exactly belongs in this list.  Though maybe this is obvious if you have actually done this ever, and my ignorance is the problem!

Reply:
Great call -- adding now